src/modules/Builder/state/Themes/Themes.reducer.ts

```diff
@@ -1,19 +1,30 @@
 import { ActionReducerMapBuilder } from '@reduxjs/toolkit';
+import uniqBy from 'lodash.uniqby';
 
-import { getPendingData, getFulfilledData, getRejectedData } from 'shared/utils/state';
+import { getPendingData, getRejectedData } from 'shared/utils/state';
 
 import { state as initialState } from './Themes.state';
 import { ThemesSchema } from './Themes.schema';
 import { getThemes } from './Themes.thunk';
 
+export const reducers = {
+  resetThemes: (state: ThemesSchema): void => {
+    state.themes = initialState.themes;
+  },
+};
+
 export const extraReducers = (builder: ActionReducerMapBuilder<ThemesSchema>): void => {
   getPendingData({ builder, thunk: getThemes, key: 'themes' });
-  getFulfilledData({
-    builder,
-    thunk: getThemes,
-    key: 'themes',
-    initialState,
+
+  builder.addCase(getThemes.fulfilled, ({ themes }, { payload }) => {
```

Comment:
maybe create function in shared/utils/state and use it also for notifications (alerts)?